en/appendices/3-2-migration-guide.rst

```diff
@@ -11,12 +11,46 @@ CakePHP 3.2 requires at least PHP 5.5. By adopting PHP 5.5 we can provide better
 Date and Time libraries and remove dependencies on password compatiblity
 librarires.
 
+Carbon Replaced with Chronos
+============================
+
+The Carbon library has been replaced with cakephp/chronos. This new library is
```

Comment:
I feel like ``cakephp/chronos`` should link to somewhere. Not sure if it should link to the GH repo or time.rst, though.